frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -306,6 +306,9 @@ function ReaderFooter:init()
     if not self.settings.toc_markers_width then
         self.settings.toc_markers_width = DMINIBAR_TOC_MARKER_WIDTH
     end
+    if not self.settings.progress_bar_min_width_pct then
+        self.settings.progress_bar_min_width_pct = 20  -- unscaled_size_check: ignore
```

Comment:
I think you don't need that `-- unscaled_size_check: ignore` anymore (.ci/check.sh checks for `width =`, `width_pct =` shouldn't trigger it).